code/code.py

```diff
@@ -0,0 +1,21 @@
+# The program checks whether a number is an Armstrong number or not
```

Comment:
instead of printing true or false, can we make this a function named `armstrong` that takes a number and returns true or false? That will make writing a test as simple as 

`import armstrong from code.code` 

when you setup the `__init__.py`

Reference https://portingguide.readthedocs.io/en/latest/imports.html

Reply:
Ok understood. I'll make the changes ✌🏻

Reply:
Kindly check and suggest other changes if required. If everything looks fine, merge it to the master.